### PR TITLE
Create cat.json

### PR DESCRIPTION
**Pull request by xxCopyCat**
Hope this gets accepted... Oh, and I have a preview:
https://kumoneo.lol

### CODE REVIEW

**Review by DEV-DIBSTER — DISMISSED** [review]

File: sub-logs/cat.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "repo": "",
+
+    "owner": {
+      "username": "xxCopyCat",
+      "email": "ibrahimbode47@gmail.com"
+    },
+
+    "target": {
+      "CNAME": {
+         "name": "CNAME",
```
Comment (DEV-DIBSTER) [inline sub-logs/cat.json:11]: This should be the same as the file name.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: sub-logs/cat.json (hunk 1 above)
Comment (xxCopyCat) [reply sub-logs/cat.json:11]: Ah alright got it... Do I create a new pull request? I'm not sure how to manage with github.

**Comment by Tweak4141** [comment]
Just edit the file via the web Ui

**Comment by Tweak4141** [comment]
Are you sure you wish to proxy though cloudflare? 99% of the time it leads to issues.

**Comment by xxCopyCat** [comment]
Edited the file.

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by xxCopyCat** [comment]
And yeah I'm sure of it.

**Pull request merged**